fuzz/fuzz_targets/rt.rs

```diff
@@ -0,0 +1,12 @@
+#![no_main]
```

Comment:
Would like this file name to be more descriptive unless it's some kind of convention I don't know about.